src/test/java/com/gooddata/report/ReportExportFormatTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2004-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.report;
+
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+
+import static com.gooddata.report.ReportExportFormat.toExportFormat;
+import static org.hamcrest.MatcherAssert.*;
```

Comment:
We can do star-imports now? :)

Reply:
yes, we can ;) we do use default idea code style